app/src/bcsc-theme/navigators/OnboardingStack.tsx

```diff
@@ -32,7 +45,7 @@ const OnboardingStack = (): JSX.Element => {
         headerShadowVisible: false,
         headerBackTestID: testIdWithKey('Back'),
         headerBackAccessibilityLabel: t('Global.Back'),
-        header: createHeaderWithoutBanner,
+        header: createHeaderWithBanner(handleManageDevices),
```

Comment:
nit: createHeaderWithBanner will inject the header with the notification banners for all screens in this stack.

Intentionally using createHeaderWithoutBanner to prevent those from appearing. 

We probably should include a comment about this, and revert back to createHeaderWIthoutBanner